mappings/net/minecraft/UrlConstants.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_goxaugqo net/minecraft/UrlConstants
```

Comment:
ohhh, that's a neat alternative pattern that `ScreenTexts` could be renamed to; Hm, I do wonder if `TextConstants` would fit though;
anyway, yeah, i don't like `ScreenTexts` and Mojmap's `CommonComponents` is close to being better; therefore, it should be renamed and it should share a theme with this

oh, about the package, hm, i do wonder if it should be at `util`

Reply:
it's at net.minecraft.util now (even though SharedConstants isn't)